participants/Borodina/pr_tutorial/simple_functions.py

```diff
@@ -1,6 +1,10 @@
 def fibonacci(max):
+    if type(max) != int:
+        raise ValueError('max value should be int')
```

Comment:
Maybe spell out integer in full (sorry I just wanted to try out changing something xD).
```suggestion
        raise ValueError('max value should be integer')
```